programs/server/users.xml

```diff
@@ -5,6 +5,15 @@
     <profiles>
         <!-- Default settings. -->
         <default>
+            <!-- How to choose between replicas during distributed query processing.
+                 random - choose random replica from set of replicas with minimum number of errors
+                 nearest_hostname - from set of replicas with minimum number of errors, choose replica
+                  with minimum number of different symbols between replica's hostname and local hostname
+                  (Hamming distance).
+                 in_order - first live replica is chosen in specified order.
+                 first_or_random - if first replica one has higher number of errors, pick a random one from replicas with minimum number of errors.
+            -->
+            <load_balancing>random</load_balancing>
```

Comment:
why do we need this?

Reply:
will remove it. It's probably added by accident when rebase